model/Readme.md

```diff
@@ -0,0 +1 @@
+Github does not allow files larges than 100MB, please [download](https://drive.google.com/file/d/1aXA3NvXf9SIYCUDM9AnNxRXUeGATcLVz/view?usp=sharing) the model first. 
```

Comment:
Model for running inference independently: https://drive.google.com/file/d/12HOk5H8tCuhtiKweG_pIECDYAwQ4c_Vr/view?usp=sharing
Model with lower false positives: https://drive.google.com/file/d/1EHf2SOf9GbdeSGCgOsH1Y0BpHgqY1WW_/view?usp=sharing